meson.build

```diff
@@ -500,6 +500,7 @@ if get_option('enable-test')
   testenv.set('NNSTREAMER_DECODERS', path_nns_plugin_decoders)
   testenv.set('NNSTREAMER_CONVERTERS', path_nns_plugin_converters)
   testenv.set('NNSTREAMER_SOURCE_ROOT_PATH', meson.source_root())
+  testenv.set('NNSTREAMER_BUILD_ROOT_PATH', meson.build_root())
```

Comment:
is it necessary to update below files for testenv ?

 tests\nnstreamer_filter_extensions_common\meson.build
 tests\nnstreamer_filter_mvncsdk2\meson.build 
 tests\tizen_capi\meson.build

Reply:
Yes.. or they may rename their env variable name.

Reply:
@myungjoo @jaeyun-jung Acked.